### PR TITLE
LPS-110416

### PR DESCRIPTION
**Pull request by tinatian**

### CODE REVIEW

**Comment by tinatian** [comment]
ci:test:relevant

File: modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/reference/TableReferenceDefinitionManager.java [hunk 1]
```diff
@@ -167,15 +167,7 @@ private Long _getClassNameId(TableReferenceInfo<?> tableReferenceInfo) {
 						currentTableReferenceDefinition.getTable(),
 						key -> new ArrayList<>());
 
-				for (TableJoinHolder currentParentJoinHolder :
-						currentParentJoinHolders) {
-
-					combinedChildJoinHolders.add(
-						new TableJoinHolder(
-							currentParentJoinHolder.getJoinPKColumn(),
```
Comment (Preston-Crary) [inline modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/reference/TableReferenceDefinitionManager.java]: Hi @tinatian,

this is swapping the join/from columns.

Comment (tinatian) [reply modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/reference/TableReferenceDefinitionManager.java]: Aha, I see... Please ignore this change.

Tina.

**Comment by Preston-Crary** [comment]
Please see https://github.com/tinatian/liferay-portal/pull/1773

**Pull request closed without merging**